{"name": "test_div[get_div_datas3]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1.5 == 0.1", "trace": "self = <test_calc_new.TestCalc object at 0x7f25447e0610>, get_calc = <pythoncode.calculator.Calculator object at 0x7f25447b1310>\nget_div_datas = [0.3, 0.2, 0.1]\n\n    @pytest.mark.run(order=4)\n    # 测试div函数\n    def test_div(self, get_calc, get_div_datas):\n        try:\n            # 调用div函数，返回的结果保存在result里面\n            result = get_calc.div(get_div_datas[0], get_div_datas[1])\n            if isinstance(result, float):\n                result = round(result,2)\n        except Exception as e:\n            print(e)\n        #判断result结果是否等于期望的值\n>       assert result == get_div_datas[2]\nE       assert 1.5 == 0.1\n\ntest_calc_new.py:33: AssertionError"}, "attachments": [{"name": "stdout", "source": "503fa692-eb00-4d6f-93de-1a7009452935-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "get_div_datas", "value": "[0.3, 0.2, 0.1]"}], "start": 1609517343048, "stop": 1609517343048, "uuid": "ad32bbaf-f726-4d81-b6d1-78a04f845304", "historyId": "5a5fc454f2e095c474c410d1201f3db2", "testCaseId": "b433a02ec13245ef6dd3b014f7c89e96", "fullName": "pytestwork2.test_calc_new.TestCalc#test_div", "labels": [{"name": "tag", "value": "run(order=4)"}, {"name": "parentSuite", "value": "pytestwork2"}, {"name": "suite", "value": "test_calc_new"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "ygl-virtual-machine"}, {"name": "thread", "value": "16108-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "pytestwork2.test_calc_new"}]}